nservicebus/recoverability/configure-error-handling.md

```diff
@@ -16,13 +17,13 @@ related:
 
 ## Configure the error queue address
 
-When a message fails NServiceBus [automatically retries](/nservicebus/recoverability/configure-immediate-retries.md) the message. On repeated failure NServiceBus forwards that message to a designated error queue.
+When message processing fails NServiceBus performs automatic [delivery retries](/nservicebus/recoverability). When that fails as well NServiceBus forwards failing message to a designated error queue.
```

Comment:
> automatic delivery retries 

I actually prefer "automatic retries" but you could argue that delivery retries could be more accurate. But if the transport actually does not roll back to the queue and  "delivery retries" might not be more accurate. Automatic retries covers all those cases because it doesn't imply redelivery directly.